src/main/resources/reference.conf

```diff
@@ -321,6 +330,32 @@ resourceTypes = {
     }
     reuseIds = true
   }
+  persistent-disk = {
+    actionPatternObjects = {
+      read = {
+        description = "read metadata and contents of persistent disk"
+      }
+      attach = {
+        description = "attach persistent disk to a VM"
+      }
+      modify = {
+        description = "modify persistent disk"
+      }
+      delete = {
+        description = "delete persistent disk"
+      }
+      read_policies = {
+        description = "view all policies and policy details for the persistent disk"
+      }
+    }
+    ownerRoleName = "creator"
+    roles = {
+      creator = {
+        roleActions = ["read", "attach", "modify", "delete", "read_policies"]
+      }
+    }
+    reuseIds = true
```

Comment:
I don't think you want `reuseIds = true`. This would allow someone to delete the resource in sam then allow another to recreate the resource with the same id and take control of the persistent disk

Reply:
Gotcha, thanks. The `notebook-cluster` resource has `reuseIds = true`, is that something I should change as well then?

Reply:
fwiw Leo now uses UUIDs for sam resource IDs. So I think we could remove `reuseIds = true` for `notebook-cluster` as well?

Reply:
if you are using UUIDs then `reuseIds=false` is recommended as you never intend to reuse anyway